action.yml

```diff
@@ -235,8 +235,8 @@ runs:
           ./cosign verify-blob --key $RELEASE_COSIGN_PUB_KEY --signature ${desired_cosign_filename}.sig cosign_${{ inputs.cosign-release }}
 
           rm cosign
-          mv cosign_${{ inputs.cosign-release }} ${cosign_executable_name}
-          chmod +x ${cosign_executable_name}
+          sudo mv cosign_${{ inputs.cosign-release }} ${cosign_executable_name}
+          sudo chmod +x ${cosign_executable_name}
```

Comment:
thanks @saschagrunert, is it possible to add a test job to test this use case? thanks!

Reply:
Done, added a test case for that.

Reply:
<3